flume-ng-channels/flume-kafka-channel/src/main/java/org/apache/flume/channel/kafka/KafkaChannel.java

```diff
@@ -76,8 +77,32 @@
 import java.util.concurrent.atomic.AtomicBoolean;
 import java.util.concurrent.atomic.AtomicReference;
 
-import static org.apache.flume.channel.kafka.KafkaChannelConfiguration.*;
```

Comment:
I believe our style guide allows wildcard imports for static fields

Reply:
Reverted the change

Reply:
Is this still outstanding?